mern-ecommerce/controllers/category.js

```diff
@@ -68,3 +68,36 @@ exports.list = (req, res) => {
     res.json(data);
   });
 };
+
+exports.addProductToCategory = (req, res) => {
```

Comment:
The function `addProductToCategory` is not checking if the product is already in the category's products list before adding it. This could lead to duplicate products in the category. It would be better to check if the product is already in the list before adding it.